app/oxalis/security/WkSilhouetteEnvironment.scala

```diff
@@ -24,20 +25,34 @@ trait WkEnv extends Env {
 class WkSilhouetteEnvironment @Inject()(conf: WkConf,
                                         tokenDAO: TokenDAO,
                                         userService: UserService,
-                                        val messagesApi: MessagesApi
+                                        cookieHeaderEncoding: CookieHeaderEncoding
                                      )(implicit val executionContext: ExecutionContext) extends Environment[WkEnv] {
   val eventBusObject = EventBus()
-  val cookieSettings = conf.raw.underlying.as[CookieAuthenticatorSettings]("silhouette.cookieAuthenticator")
+
+  val cookieSettings = CookieAuthenticatorSettings(
+    conf.Silhouette.CookieAuthenticator.cookieName,
+    conf.Silhouette.CookieAuthenticator.cookiePath,
+    None,
+    conf.Silhouette.CookieAuthenticator.secureCookie,
+    conf.Silhouette.CookieAuthenticator.httpOnlyCookie,
+    Some(Cookie.SameSite.Lax),
+    conf.Silhouette.CookieAuthenticator.useFingerprinting,
+    Some(conf.Silhouette.CookieAuthenticator.cookieMaxAge.toMillis millis),
+    None,
+    conf.Silhouette.CookieAuthenticator.authenticatorExpiry.toMillis millis
+  )
```

Comment:
Can’t pass a raw config object anymore because a new parameter has type parameters.
We could possibly hard-code some of these values here as some future refactoring, there isn’t a good reason for having them in the config anymore.